runconfig/opts/parse.go

```diff
@@ -204,6 +206,17 @@ func Parse(cmd *flag.FlagSet, args []string) (*container.Config, *container.Host
 		}
 	}
 
+	var maxIOBandwidth int64
```

Comment:
int64 because the unit parsing via RAMInBytes doesn't output uint64. I'd considered writing a RAMInBytes uint64 version, but it is in vendored go-units.